Python/optimizer_bytecodes.c

```diff
@@ -467,6 +467,15 @@ dummy_func(void) {
         res = sym_new_truthiness(ctx, value, false);
     }
 
+    op(_UNARY_INVERT, (value -- res)) {
+        if (sym_matches_type(value, &PyLong_Type) || sym_matches_type(value, &PyBool_Type)) {
+            res = sym_new_type(ctx, &PyLong_Type);
```

Comment:
Similar suggestion here as in the other PR:

```suggestion
        PyTypeObject *type = sym_get_type(value);
        if (value == &PyLong_Type || value == &PyBool_Type) {
            res = sym_new_type(ctx, &PyLong_Type);
```